lib/cn/cn_tree.c

```diff
@@ -1735,6 +1745,19 @@ cn_comp_commit(struct cn_compaction_work *w)
     if (split)
         cn_split_nodeids_get(w, nodeidv);
 
+    /* TODO: For now, do not let split proceed until we have removed
+     * and/or fixed all the fanout related code.
+     */
+    if (w->cw_action == CN_ACTION_SPLIT) {
+        w->cw_err = merr(EAGAIN);
+        ev_debug(1);
+        goto done;
+    }
+
+    /* TODO: Failure past this point can result in an unterminated kvset
```

Comment:
I think this comment can be removed now.

Reply:
Right.  But we probably need some verbiage about what you can and cannot do in this path going forward :-)